DNN Platform/DotNetNuke.Web.Deprecated/UI/WebControls/TermsList.cs

```diff
@@ -1,173 +1,173 @@
-            }
-        }
-
-        public void BindTerms(List<Term> terms, bool isHeirarchical, bool dataBind)
-        {
-            this._IsHeirarchical = isHeirarchical;
-
-            this._ListBox.DataSource = terms;
-            this._TreeView.DataSource = terms;
-
-            if (dataBind)
-            {
-                this._ListBox.DataBind();
-                this._TreeView.DataBind();
-            }
-        }
-
-        public void ClearSelectedTerm()
-        {
-            this._ListBox.SelectedIndex = Null.NullInteger;
-            this._TreeView.UnselectAllNodes();
-        }
-
-        protected override void CreateChildControls()
-        {
-            this.Controls.Clear();
-
-            this._ListBox = new DnnListBox();
-            this._ListBox.ID = string.Concat(this.ID, "_List");
-            this._ListBox.DataTextField = "Name";
-            this._ListBox.DataValueField = "TermId";
-            this._ListBox.AutoPostBack = true;
-            this._ListBox.SelectedIndexChanged += this.ListBoxSelectedIndexChanged;
-
-            this._TreeView = new DnnTreeView();
-            this._TreeView.ID = string.Concat(this.ID, "_Tree");
-            this._TreeView.DataTextField = "Name";
-            this._TreeView.DataValueField = "TermId";
-            this._TreeView.DataFieldID = "TermId";
-            this._TreeView.DataFieldParentID = "ParentTermId";
-            this._TreeView.NodeClick += this.TreeViewNodeClick;
-
-            this.Controls.Add(this._ListBox);
-            this.Controls.Add(this._TreeView);
-        }
-
-        protected override void OnInit(EventArgs e)
-        {
-            this.EnsureChildControls();
-        }
-
-        protected override void OnPreRender(EventArgs e)
-        {
-            this._ListBox.Visible = !this.IsHeirarchical;
-            this._TreeView.Visible = this.IsHeirarchical;
-
-            this._ListBox.Height = this.Height;
-            this._ListBox.Width = this.Width;
-            this._TreeView.Height = this.Height;
-            this._TreeView.Width = this.Width;
-
-            this._TreeView.ExpandAllNodes();
-
-            base.OnPreRender(e);
-        }
-
-        protected virtual void OnSelectedTermChanged(TermsEventArgs e)
-        {
-            // Raise the SelectedTermChanged Event
-            if (this.SelectedTermChanged != null)
-            {
-                this.SelectedTermChanged(this, e);
-            }
-        }
-
-        private void ListBoxSelectedIndexChanged(object sender, EventArgs e)
-        {
-            // Raise the SelectedTermChanged Event
-            this.OnSelectedTermChanged(new TermsEventArgs(this.SelectedTerm));
-        }
-
-        private void TreeViewNodeClick(object sender, RadTreeNodeEventArgs e)
-        {
-            // Raise the SelectedTermChanged Event
-            this.OnSelectedTermChanged(new TermsEventArgs(this.SelectedTerm));
-        }
-    }
-}
+    public class TermsList : WebControl

```

Comment:
I think this one got missed, but other than that it all looks very good to me

Reply:
Just pushed that one